README.md

```diff
@@ -1,4 +1,4 @@
-# cargo-build-deps
+# cargo-build-dependenciess
```

Comment:
Thanks for the help! However, there is a minor typo.
If you change it to `cargo-build-dependencies`, I'll merge it asap.

Reply:
lol - of course